### PR TITLE
improvement(plugins/sct): Store DC Name in CloudInstanceDetails

### LINKED ISSUE #222: Please, add column "DC name" to resources list ("Resources" tab)
Please, add column "DC name" to resources list ("Resources" tab).
We need it for multi DC tests - to know in which DC the instance is located.

Example:
https://argus.scylladb.com/test/1158bb3b-c12c-4c8c-a38f-d4492c398a4d/runs?additionalRuns[]=c4c3ee91-c03b-4dc9-9df9-4e1f42885db5

### PR DESCRIPTION
**Pull request by k0machi**
This change allows Argus to track datacenter names inside
CloudInstanceDetails.

Fixes #222

### CODE REVIEW

File: argus/client/sct/client.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def submit_performance_results(self, performance_results: PerformanceResultsRequ
         self.check_response(response)
 
     def create_resource(self, name: str, resource_type: str, public_ip: str, private_ip: str,
-                        region: str, provider: str, shards_amount: int, state=ResourceState.RUNNING) -> None:
+                        region: str, provider: str, dc_name: str, shards_amount: int, state=ResourceState.RUNNING) -> None:
```
Comment (fruch) [inline argus/client/sct/client.py:151]: if we have region, that's in most cases 1to1 with dc_name, az would also be useful

Comment (fruch) [reply argus/client/sct/client.py:151]: in most cases it's the same:
region <-> datacenter
availability zone <-> rack

but then we have fake racks, and soon also fake regions
so datacenter and rack, and we need precise about where we get the information from on SCT end.

**Comment by k0machi** [comment]
image

File: argus/client/sct/client.py (hunk 1 above)
Comment (k0machi) [reply argus/client/sct/client.py]: Done

**Pull request merged**